deepinfra/utils/read_stream.py

```diff
@@ -0,0 +1,67 @@
+import io
+import base64
+import requests
+
+class ReadStreamUtils:
+    """
+    Utility class for working with files.
+    """
+
+    @staticmethod
+    def buffer_to_stream(buffer):
```

Comment:
You should check `mypy`. Python supports type annotations, just like TypeScript (but better, because it's the same language). You should add them to all methods, maybe another commit.